packages/create-bison-app/tasks/copyFiles.js

```diff
@@ -39,6 +39,17 @@ async function copyFiles({ variables, targetFolder }) {
     copyWithTemplate(fromPath("README.md.ejs"), toPath("README.md"), variables),
     copyWithTemplate(fromPath("_.gitignore"), toPath(".gitignore"), variables),
 
+    copyWithTemplate(
+      fromPath("tailwind.config.js"),
+      toPath("tailwind.config.js"),
+      variables
+    ),
+    copyWithTemplate(
+      fromPath("postcss.config.js"),
+      toPath("postcss.config.js"),
+      variables
+    ),
```

Comment:
Can these files be moved down to be copied by the `cpy` function: https://github.com/echobind/bisonapp/blob/canary/packages/create-bison-app/tasks/copyFiles.js#L89

The `copyWithTemplate` is only necessary to copy EJS templates.